working-examples/breadcrumb-trail/index.html

```diff
@@ -3,34 +3,48 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">    
-    <title>G65 Example 3: Breadcrumb Example</title>
+    <title>G65 Proviging a breadcrumb trail examples</title>
```

Comment:
Is this a typo? Proviging?

Reply:
```suggestion
    <title>G65 Providing a breadcrumb trail examples</title>
```

Reply:
"Proviging" isn't a word where you live? Sheesh!